src/server/views/partials/team-comment.handlebars

```diff
@@ -1,11 +1,11 @@
 <div class="bo-comment-wrap">
     <div class="row">
         <div class="col-xs-2">
-            <a href="{{getImageByWidth 800 user.profilePic.sizes}}" data-lightbox="image-{{id}}-{{user.id}}"
+            <a href="{{user.profilePic.url}}" data-lightbox="image-{{id}}-{{user.id}}"
```

Comment:
how are different sizes requested after the update?

Reply:
Different sizes can be requested via cloudinary by modifing the url https://cloudinary.com/cookbook/resize_an_image

We can do this, but we should take care of how many transformations we want to use (which is why I decided to skip this part for now and create a useful strategy for it later)

https://github.com/BreakOutEvent/breakout-frontend/issues/301